package.json

```diff
@@ -228,7 +228,12 @@
     }
   },
   "lint-staged": {
-    "*.{js,json,md,graphql}": [
+    "*.{js}": [
+      "npm run lint:fix",
```

Comment:
I Inspected to understand what could be wrong, here is the mistake, when you run `npm run lint:fix` this command will run ` eslint '**/*.js' "--fix"` along with the staged file paths. 

`'**/*.js'` will include all `.js` files as it was intended for the script command `npm run lint`.

The solution is to this is to use `eslint` command directly i.e 
```json
"*.{js}": [
      "eslint --fix",
      "prettier --write",
      "git add"
    ],
```